internal/stage7.go

```diff
@@ -64,8 +70,20 @@ func testType2(stageHarness *test_case_harness.TestCaseHarness) error {
 
 	for _, executable := range nonAvailableExecutables {
 		command := fmt.Sprintf("type %s", executable)
-		actualPath := getPath(executable)
-		expectedPattern := fmt.Sprintf(`^(bash: type: )?%s\r\n`, actualPath)
+
+		var commandOutput string
+		if executable == "my_exe" {
+			commandOutput = customExecutablePath
+		} else {
+			path, err := exec.LookPath(executable)
```

Comment:
Don't think this is required, we're only checking for "not found" cases here no?